File: src/fiefapp.svelte/src/models/cargo.ts
interface ICargo {
  id: string;
  silver: number;
  base: number;
  luxury: number;
  wood: number;
  iron: number;
  stone: number;
  other: number;
  otherInformation: string;
}

export class Cargo implements ICargo {
  id: string;
  silver: number;
  base: number;
  luxury: number;
  wood: number;
  iron: number;
  stone: number;
  other: number;
  otherInformation: string;

  constructor() {
    this.id = '';
    this.silver = 0;
    this.base = 0;
    this.luxury = 0;
    this.wood = 0;
    this.iron = 0;
    this.stone = 0;
    this.other = 0;
    this.otherInformation = '';
  }
}

File: src/fiefapp.entities/Cargo.cs
using fiefapp.entities.Interfaces;
using System;
using System.Collections.Generic;

namespace fiefapp.entities
{
    public class Cargo : IBaseEntity
    {
        public Cargo()
        {
            Id = Guid.NewGuid();
            Notes = new HashSet<string>();
        }

        public Guid Id { get; set; }
        public int Silver { get; set; }
        public int Base { get; set; }
        public int Luxury { get; set; }
        public int Iron { get; set; }
        public int Stone { get; set; }
        public int Wood { get; set; }
        public ICollection<string> Notes { get; set; }
    }
}
